src/user-guide/writing-workflows/runtime.rst

```diff
@@ -115,18 +115,6 @@ inheritance in a workflow. You can extract specific items, e.g.:
    $ cylc config --item '[runtime][var_p2]script' inherit.multi.one
    echo "RUN: run-var.sh"
 
-or use the ``--sparse`` option to print entire namespaces
-without obscuring the result with the dense runtime structure obtained
-from the root namespace:
```

Comment:
Do we not want to document the new `--defaults` option somewhere?

Reply:
It's documented in the CLI but a fairly obscure option.